### DISCUSSION TITLE
使用 R 抓取网页数据--基于国务院政策文件库 | /project/r-crawl-data/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#623] [Announcements]
https://yuanfan.rbind.io/project/r-crawl-data/

**Comment by botsp** [comment]
看到一个刘晓博发布的"体制内强度指数"的分析，看到就想来讲下，感觉是很有意思的分析角度，对行政与经济数据。

**Reply by earfanfan** [reply]
我去搜了一下“体制内强度指数”，这个指标用于对比确实是有点意思的，但是作者要强行按33%来划定一个标准线就不那么有意思了。

话说去年我还看过公积金的数据，当时对比了北京、上海、武汉按缴存单位性质划分的公积金数据，如下。像武汉的事业单位和国企加起来超过40%了。
![](https://yuanfan.rbind.io/images/2023/2023-04-08-11.png)

从我看过的一些数据和书籍来看，94年分税制改革后，城市才开始推行公积金制度，由于历史原因最开始确实是国企事业单位的体量更大，自然缴存公积金占比更高，随着时间推移，人口从农村向城市迁移，受到分税制改革影响大多数中西部省份都是举一省之财力发展省会城市，这样国企事业单位的公积金缴存占比开始下降。至于是否缴存公积金涉及很多复杂因素，但国企事业单位缴存占比在不同情况下作用不一样，比如在人口流入的大城市，经济效益稳定的国企可能对城市经济起到一种基石的作用，是正向吸引力；但是对人口流出的小城市，确实也会如同那位刘晓博分析的一样，可能代表着民营经济没有发展起来。但也无法排除其他可能，假如当地的工业制造业企业为城市创造了最大比例的生产值，如果较少为工人缴纳公积金，但是交税方面并不藏着，当地政府利益相关可能也不会非要强迫企业为员工缴公积金。

**Reply by botsp** [reply]
明白了，似乎是国企较早的参与公积金制度，而后来发展起来的私营部门往往又不参与这个公积金制度。不能简单从公积金这个指标来判断这个"体制内"指数、也不能简单认为"体制内"就是经济发展的消极因素，可以这样理解不。
感觉社保规范化也就是最近这些年的事情，私营部门做好社保就挺难了，交公积金更是抗拒。

**Reply by earfanfan** [reply]
我觉得大概可以这样理解吧。

公积金制度发展这么多年，也是有不同的历史时期的，好些年以前提取公积金很难，有些人可能会因为这点而选择不缴存公积金。现在公积金提取变得容易，但私企仍然交的少的话，我推测并不是全都出于抗拒。几年前我曾经在一家几十人的小公司待过，那家公司会以当时城市最低工资水平来帮忙缴纳最低比例的社保公积金，但实际发放的工资还是按谈好的来，如果用这种方法来缴的话，其实对小公司的成本不会增大多少。所以我猜，是就业人数这个分母的一些隐藏问题，有可能大多数私企是更小规模，连可以这么操作的财务会计出纳的员工都没得。

**Reply by botsp** [reply]
确实有研究[thumb]
